app/components/pages/Help.vue.ts

```diff
@@ -5,6 +5,63 @@ import electron from 'electron';
 @Component({
 })
 export default class Help extends Vue {
+  videos = [
+    {
+      readMore: false,
+      name: 'Streamlabs OBS Quickstart',
```

Comment:
i think any user-visible copy should be wrapped in `$t()`

Reply:
ok - even though YT videos/titles are english only?

Reply:
oh hm 🤔 im not actually sure in that case lol

Reply:
yeah I don't know either, don't want to be misleading